fieldservice_maintenance/models/fsm_equipment.py

```diff
@@ -19,13 +20,18 @@ class FSMEquipment(models.Model):
 
     @api.model
     def create(self, vals):
+        maintenance_team_id = self.env["maintenance.team"].search(
+            [("company_id", "=", self.env.company.id)], limit=1
```

Comment:
issue (blocking): maintenance.team may have no company_id

```suggestion
            [("company_id", "in", (self.env.company.id, False)], limit=1
```